submissions/cumiOyemike/database/init.sql

```diff
@@ -0,0 +1,124 @@
+/* create database */
+CREATE DATABASE infinity_sparkles;
+SHOW DATABASES;
+
+/* use the created infinity_sparkles database*/
+USE infinity_sparkles;
+
+/*create table for product categories*/
+CREATE TABLE product_categories (
+    id INT NOT NULL AUTO_INCREMENT,
+    category VARCHAR(50) NOT NULL,
+    created_at DATETIME NOT NULL,
+    PRIMARY KEY (id)
+);
+/*inserting values into the product_categories table*/
+INSERT INTO product_categories (category, created_at)
+VALUES ('Earrings', now()),
+('Necklaces', now());
+
+/*create table for admin users*/
+CREATE TABLE admin_users (
+    id INT NOT NULL AUTO_INCREMENT,
+    name VARCHAR(100) NOT NULL,
+    email VARCHAR(40) NOT NULL,
+    password VARCHAR(20) NOT NULL,
+    phone_number VARCHAR(20) NOT NULL,
+    created_at DATETIME NOT NULL,
+    PRIMARY KEY (id)
+);
+/*inserting values into the admin_users table*/
+INSERT INTO admin_users (name, email, password, phone_number, created_at)
+VALUES ('Shola Daniel', 'sholadan@mail.com', 'ewfwerge66', '08123456789', now()),
+('Ada Chukwuma', 'adachuks@mail.com', 'rr54efffef', '09123456789', now()),
+('Hannah Adebayo', 'hanbayo@mail.com', 'gfegtgt', '07123456789', now());
+
+/*create table for registered customers*/
+CREATE TABLE registered_customers (
+    id INT NOT NULL AUTO_INCREMENT,
+    first_name VARCHAR(50) NOT NULL,
+    last_name VARCHAR(50) NOT NULL,
+    email VARCHAR(40) NOT NULL,
+    password VARCHAR(20) NOT NULL,
+    customer_address_id INT NOT NULL, /* address from customer_addresses table*/
```

Comment:
This should not be here. Instead, the `registered_customer_id` should be a foreign key on `contact_addresses` table

Reply:
Hello! Thanks for the correction. I have made the changes. Kindly review agian.